pubspec.yaml

```diff
@@ -39,3 +40,9 @@ dev_dependencies:
   http: '^0.11.0'
   js: '^0.6.0'
   scheduled_test: '^0.12.5'
+
+dependency_overrides:
```

Comment:
did you mean to check this in (e.g. so local developers always test against git?)